src/main/java/io/vertx/core/Vertx.java

```diff
@@ -220,6 +220,13 @@ static void clusteredVertx(VertxOptions options, Handler<AsyncResult<Vertx>> res
    */
   DnsClient createDnsClient(int port, String host);
 
+  /**
+   * Create a DNS client to connect to automatically chosen DNS server
```

Comment:
Create a DNS client to connect to the DNS server configured by {@link VertxOptions#getAddressResolverOptions()}

Reply:
changed